CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 <!-- ## [Unreleased] -->
```

Comment:
Why do you include this line in all the changelogs?

Reply:
We could (theoretically) uncomment the line and collect all unreleased changes below it. Then, when releasing, we can just move the changes down to an actual new release. I wish I'd remember to do that 🐤